### DISCUSSION TITLE
Export multiple languages as one file?

### DISCUSSION THREAD

**Discussion by egorkaway** [tolgee/tolgee-platform#2426] [Q&A] [answered]
Is there a way to set up export so that just one file is exported, not dozens? 

We could really benefit from something like this instead of 1 file per language: 

```
{
  "en": {
    "true": "Yes"
  },
  "es": {
    "true": "Sí"
  },
```

JSON is not critical, we can work with any other format, but one file per project that contains all languages versus 1 file per language as right now - is. 
Please advise.

**Comment by JanCizmar** [comment, marked as answer]
Hey! Sorry this is not supported now. The only suggestion I can give you is to use the CLI for the export and then merge the json files into one using custom script. It should now be hard to write (ChatGPT can help 🤠).
